src/main/java/org/AdvancedJavaEindopdracht/ConvertToDTO.java

```diff
@@ -0,0 +1,18 @@
+package org.AdvancedJavaEindopdracht;
+
+import org.AdvancedJavaEindopdracht.resource.dto.UserDTO;
+import org.AdvancedJavaEindopdracht.resource.model.User;
+
+
+public class ConvertToDTO {
```

Comment:
We moeten even binnen de groep overleggen hoe we het 'mappen' gaan doen. Ik doe dit op een iets andere manier die IMO iets mooier is :) haha